protobufs/livekit_analytics.proto

```diff
@@ -230,6 +230,7 @@ message FeatureUsageInfo {
   enum Feature {
     KRISP_NOISE_CANCELLATION = 0;
     KRISP_BACKGROUND_VOICE_CANCELLATION = 1;
+    AIC_AUDIO_ENHANCEMENT = 2;
```

Comment:
Is it generic like this or specific like `NOISE_CANCELLATION`, `BVC`?

Reply:
`audio enhancement` is what they use everywhere where I would expect krisp to use `noise cancellation` . 

We could be more specific by specifying the model name? 
`AIC_AUDIO_ENHANCEMENT_QUAIL` ? 

for reference https://github.com/ai-coustics/aic-sdk-py/blob/main/README.md